tests/diffDataParent.js

```diff
@@ -0,0 +1,95 @@
+require([
+    'chai',
+    'jquery',
+    'underscore',
+    'tests/utils',
+    'text!static/diffDataParent/parse.xml'
+], function (
+    chai,
+    $,
+    _,
+    util,
+    PARSE_XML
+) {
+    var assert = chai.assert,
+        call = util.call;
+
+    describe("Control elements with different data parents", function() {
+        before(function (done) {
+            util.init({
+                javaRosa: {langs: ['en', 'hin']},
```

Comment:
What do you think of removing one of these languages? I don't believe the features being tested here have any impact on multi-language forms, and it would reduce the size of test XML and possibly make the tests a bit faster.